src/frontend/src/pages/HomePage/UsefulLinks.tsx

```diff
@@ -69,12 +82,37 @@ const UsefulLinks: React.FC = () => {
       <Settings sx={{ fontSize: 17, color: theme.palette.text.primary }} />
       <Link
         href="https://nerdocs.atlassian.net/wiki/spaces/NER/pages/4554841/Hardware+Guidelines"
+        target="blank"
         underline="hover"
         fontSize={19}
         sx={{ pl: 1 }}
       >
         Hardware Guidelines
       </Link>
```

Comment:
Sorry i didnt pull when I looked at the new thing when you merged you added it back in

Reply:
I think it should be fixed now